python/pyspark/ml/tests.py

```diff
@@ -655,6 +656,20 @@ def test_nested_pipeline_persistence(self):
             except OSError:
                 pass
 
+    def test_write_property(self):
+        lr = LinearRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
+        lr.write.save(lr_path)
```

Comment:
I think self.assertTrue(isinstance(lr.write, MLWriter)) should be good enough as lr is the inheriting class of MLWriter, which is different from my original test (in that case I directly create an MLWriter instance, which is insufficient test). I change the test case to the simpler one. Thanks!